<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePartNumberForm</class>
 <widget class="QDialog" name="ChangePartNumberForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>225</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Change the Assembly Structure</string>
  </property>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Close the form, any unsaved changes will be lost</string>
   </property>
   <property name="text">
    <string>Cancel / Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string>Save the updated assembly changes, then clear the form</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="new_part_number_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>To New Part Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="old_pn_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>From Old Part Number</string>
   </property>
  </widget>
  <widget class="ErrorFrame" name="old_part_number_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="LineEdit" name="old_part_number_edit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>336</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Required: Enter the existing Assembly to change, 1 to 15 Characters</string>
    </property>
   </widget>
  </widget>
  <widget class="ErrorFrame" name="new_part_number_frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>340</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="LineEdit" name="new_part_number_edit">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>336</width>
      <height>36</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Required: Enter the new Assembly, 1 to 15 Characters</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ErrorFrame</class>
   <extends>QFrame</extends>
   <header>lbk_library/gui/error_frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lbk_library/gui/line_edit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>old_part_number_edit</tabstop>
  <tabstop>new_part_number_edit</tabstop>
  <tabstop>change_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
